DBSetup/createDB.sql

```diff
@@ -55,6 +55,7 @@ AuthorityId SERIAL,
 Username text UNIQUE,
 Password text,
 AuthorityName text, 
+AuthorityEmail text,
```

Comment:
It looks like these changes were triplicated. I think this change is shared in #72 and #75